ELM_CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Elm Changelog
 
+## [7.0.0]
```

Comment:
Shouldn't this be a major npm package version upgrade instead of elm?

Reply:
It's a major update for both packages. Because we added another variant to the `Aggregation` type.

But yea, we also need to bump the JS package and this changelog definitely needs to be moved to the NPM_CHANGELOG.